extensions-core/stats/src/main/java/org/apache/druid/query/aggregation/variance/sql/BaseVarianceSqlAggregator.java

```diff
@@ -137,21 +149,77 @@ public Aggregation toDruidAggregation(
     );
   }
 
+  private static class VarPopSqlAggFunction extends SqlAggFunction
```

Comment:
can you add a note at the top that these various aggregate functions are same as SqlAvgAggFunction except that these versions pass `OperandTypes.Any` while standard function in calcite pass `OperandTypes.Numeric`

Reply:
Changed this to use aggregatorBuilder and added a comment about this